libs/template/presets/storefront/experience/pages/article-page.ts

```diff
@@ -0,0 +1,25 @@
+import { StaticComponent } from '@spryker-oryx/experience';
+
+export const articlePage: StaticComponent = {
+  type: 'Page',
+  meta: {
+    title: 'Article Page',
+    route: '/article/:id',
+    description: 'Default Article Page Description',
```

Comment:
we should – going forward – have those being generated from the content

Reply:
agree

Reply:
But I guess this is should be part of article component when we implement it